python/katana/lonestar/analytics/calculate_degree.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
```

Comment:
This commit also contains the changes from d78f868c6eddacec482616e44e53cd2bc4bf36b6. Please rebase this PR on top of the current katana/master so it only contains the new changes.

(I think the rebase before you commit would actually remove the duplication, but it's good to learn to rebase and to create clean PRs that don't have extra changes that already exist on master.)